integration-tests/fake_spine/fake_spine/vnp_test_responses.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from fake_spine import vnp_request_matcher_wrappers
+from fake_spine.request_matching import MatcherAndResponses
+from fake_spine.vnp_spine_response import VnpSpineResponseBuilder
+
+ASYNC_RELIABLE_INBOUND_HEADERS = {
+    'Soapaction': 'urn:nhs:names:services:psis/MCCI_IN010000UK13'
+}
+
+ASYNC_EXPRESS_INBOUND_HEADERS = {
+    'Soapaction': 'urn:nhs:names:services:psisquery/QUPC_IN160102UK05'
+}
+
+
+def vnp_test_responses() -> List[MatcherAndResponses]:
+    response_mappings = []
+
+    # TODO: make the 202 response code default?
```

Comment:
I think you did that (see SpineResponseBuilder)

Reply:
TODONE